distributed-process-tests/src/Control/Distributed/Process/Tests/CH.hs

```diff
@@ -1369,6 +1372,116 @@ testExitRemote TestTransport{..} = do
   takeMVar supervisedDone
   takeMVar supervisorDone
 
+testRegistryMonitoring :: TestTransport -> Assertion
+testRegistryMonitoring TestTransport{..} = do
+  localNode <- newLocalNode testTransport initRemoteTable
+  remoteNode <- newLocalNode testTransport initRemoteTable
+  return ()
```

Comment:
[nitpick] ?

Reply:
About...? 😂

Reply:
ops.. is `return ()` really needed there?

Reply:
No idea, it's in the original tests, which I'm pretty sure you wrote! :D

I don't think it's actually needed, no, so I'll go remove it and we'll see if the tests remain stable. They should, I mean, we're not exactly deterministic about giving up time slices so this expression shouldn't make any difference.

Reply:
Hah.. maybe it makes use of the tracing and overrived `>>=`. Ok lets keep that.